psl/psl-core/src/configuration/configuration_struct.rs

```diff
@@ -102,7 +81,14 @@ impl Configuration {
         Ok(())
     }
 
-    pub fn resolve_datasource_direct_urls_from_env<F>(
+    // Resolve datasource URL's for getConfig.
+    // The main reason this exists is:
+    //   - we want to error if we can't resolve direct_url
+    //   - we want to skip validation for url IF we have a direct_url
+    //
+    // For that last bit, we only do this currently because our validation errors on URL's starting
+    // with 'prisma://'. We would ideally like to do the other validations and ignore in this case.
+    pub fn resolve_datasource_url_prisma_fmt<F>(
```

Comment:
One of the methods uses `url`, the other `urls`. Accident?
```suggestion
    pub fn resolve_datasource_urls_prisma_fmt<F>(
```
Needs follow up changes in usage if accepted of course.